.chloggen/aws.yaml

```diff
@@ -0,0 +1,18 @@
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern
+component: awsxrayreceiver
+
+# A brief description of the change
+note: "Migrated awsxrayreceiver to AWS SDK v2 for better performance and future compatibility."
+
+# Mandatory: One or more tracking issues related to the change
+issues: [#37732]
```

Comment:
```suggestion
issues: [37732]
```

Reply:
This issue is causing .chloggen CI to fail to parse the yaml.